### ISSUE TITLE
ws.Drawings.AddPicture don't support jpeg/exif

### ISSUE THREAD

**Issue by superboy1984** [EPPlusSoftware/EPPlus#645]
i use epplus 6.0.4 export excel with image

`var imageInfo = new FileInfo(localImageFileName);`
`picture = ws.Drawings.AddPicture($"image_{DateTime.Now.Ticks}", imageInfo);`

then thrown exception: 
System.InvalidOperationException: No image handler for image type Jpg

then i change my code

`FileStream fileStream = new FileStream(localImageFileName, FileMode.Open, FileAccess.Read);`
`byte[] buffer = new byte[4];`
`fileStream.Read(buffer, 0, 4);`
`string magicNumberMapping = string.Join("", buffer.Select(element => element.ToString("X2")));`
`fileStream.Seek(0, SeekOrigin.Begin);`
`picture = ws.Drawings.AddPicture($"image_{DateTime.Now.Ticks}", fileStream, ePictureType.Jpg);`

then find those images of thrown exception magicNumberMapping is 'FFD8FFE1', work fine magicNumberMapping is 'FFD8FFE0'
FFD8FFE0(JPEG/JFIF),FFD8FFE1(JPEG/EXIF)

**Comment by JanKallman** [comment]
I'll have a look. Can you attach a failing image?

**Comment by JanKallman** [comment]
I can't reproduce your first issue. The image is identified by the extension of the file...
https://github.com/EPPlusSoftware/EPPlus/blob/bdfaf0aa31afecf016abe2e93f3d9d318aafe8a4/src/EPPlus/Drawing/ImageHandling/PictureStore.cs#L250
What is the path you provide in `localImageFileName`?

**Comment by JanKallman** [comment]
Thanks. The GenericImageReader seems to fail on this image. The TIFF header is correctly read, but it fails in a later stage and causes an exception. I'll investigate and provide a fix shortly.

I assume you use a non-GDI environment as this works in Windows and Linux with libgdiplus with the
If you have libgdiplus installed you can force the image handler to use System.Drawing instead:
```C#
package.Settings.ImageSettings.PrimaryImageHandler = new SystemDrawingImageHandler();
```

**Comment by JanKallman** [comment]
This is fixed in version 5.8.11 and 6.0.5
